diff --git a/node_modules/markdown-it-footnote/dist/markdown-it-footnote.js b/node_modules/markdown-it-footnote/dist/markdown-it-footnote.js
index 39a1d7f..550399a 100644
--- a/node_modules/markdown-it-footnote/dist/markdown-it-footnote.js
+++ b/node_modules/markdown-it-footnote/dist/markdown-it-footnote.js
@@ -36,7 +36,7 @@ function render_footnote_ref(tokens, idx, options, env, slf) {
     refid += ':' + tokens[idx].meta.subId;
   }
 
-  return '<sup class="footnote-ref"><a href="#fn' + id + '" id="fnref' + refid + '">' + caption + '</a></sup>';
+  return '<sup class="footnote-ref"><a href="#fn-anchor' + id + '" id="fnref-intext' + refid + '">' + caption + '</a></sup>';
 }
 
 function render_footnote_block_open(tokens, idx, options) {
@@ -56,7 +56,7 @@ function render_footnote_open(tokens, idx, options, env, slf) {
     id += ':' + tokens[idx].meta.subId;
   }
 
-  return '<li id="fn' + id + '" class="footnote-item">';
+  return '<li id="fn-anchor' + id + '" class="footnote-item">';
 }
 
 function render_footnote_close() {
@@ -71,7 +71,7 @@ function render_footnote_anchor(tokens, idx, options, env, slf) {
   }
 
   /* ↩ with escape code to prevent display as Apple Emoji on iOS */
-  return ' <a href="#fnref' + id + '" class="footnote-backref">\u21a9\uFE0E</a>';
+  return ' <a href="#fnref-intext' + id + '" class="footnote-backref">\u21a9\uFE0E</a>';
 }
 
 
